modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -114,6 +117,9 @@ For details about enrichment mode, see link:{neo4j-docs-base-uri}/cdc/current/ge
 | `aligned` \| `standard` \| `high_limit` \| `block`
 |
 Defines the store format if the database created is new.
+`high_limit` and `standard` formats are deprecated from 5.23.
```

Comment:
should this say 5.23? we didn't add the deprecation warning in the create database command until 5.26?

Reply:
The [store formats](https://neo4j.com/docs/operations-manual/5/database-internals/store-formats/)  page says they were deprecated in 5.23